### DISCUSSION TITLE
[Solved] problem with the "+" button to add tasks, which refuses to work

### DISCUSSION THREAD

**Discussion by phardouin** [kanboard/kanboard#5202] [Installation & Configuration]
Hello, I have a problem with the "+" button to add tasks, which refuses to work. I have had this problem in the past but can't remember how to fix it. I seem to remember that it is a problem of rights or server settings (OVH for my part). If anyone has an idea on this, I'm interested.

**Comment by KanjiCRC** [comment]
maybe it is related with the mod_security

**Comment by mecavito** [comment]
Tengo el mismo problema, y no solo no es posible agregar desde "+", tampoco es posible desde menu/+ Añadir nueva tarea
![imagen](https://user-images.githubusercontent.com/18124431/130848407-4fe02708-461d-4608-8539-e4abb1a31004.png)

**Comment by mecavito** [comment]
> 
> 
> maybe it is related with the mod_security

If it this case how can i resolve?, i'm not an expert, but I can access to the server

**Comment by phardouin** [comment]
@mecavito 
Check that you have the .htaccess with these parameters: 
<IfModule mod_security.c>
SecFilterEngine Off
SecFilterScanPOST Off
</IfModule>

But for some hosts this is not enough. For example at ovh.com you have to add an .ovhconfig file to the root of kanboard with http.firewall = none

**Comment by mecavito** [comment]
> 
> 
> @mecavito
> Check that you have the .htaccess with these parameters:
> 
> SecFilterEngine Off
> SecFilterScanPOST Off
> 
> But for some hosts this is not enough. For example at ovh.com you have to add an .ovhconfig file to the root of kanboard with http.firewall = none

Thank you very much, first I try to understand how it works, then I try to apply and finally I will tell you if it works and how i did

for the record its a VPS server with centos 7, and centos web panel

**Comment by mecavito** [comment]
> 
> 
> @mecavito
> Check that you have the .htaccess with these parameters:
> 
> SecFilterEngine Off
> SecFilterScanPOST Off
> 
> But for some hosts this is not enough. For example at ovh.com you have to add an .ovhconfig file to the root of kanboard with http.firewall = none

Hi, thanks again, I found the problem, it really was a mod_security problem, so
1. You can try with .htaccess, if it doesn't work
2. You can try with the php configuration (in very few cases, according to my basic knowledge), if it doesn't work
3. It all depends on the administration of your server, if you have full control of it or not, the brand and the version, for me (web control panel), there is a menu called mod security, just enable or disable the domain, with other brands you have to find where are the options for it
